test/screens/v2/screen_data/query_params_test.exs

```diff
@@ -14,12 +14,19 @@ defmodule Screens.V2.ScreenData.QueryParamsTest do
     end
 
     test "Retruns QueryParam struct when valid key/value params passed in" do
```

Comment:
Nit: Could we go ahead and fix this typo in the test description while we're at it?